spirv_reflect.h

```diff
@@ -1057,6 +1468,29 @@ inline SpvReflectResult ShaderModule::EnumerateDescriptorBindings(
   return m_result;
 }
 
+/*! @fn EnumerateEntryPointDescriptorBindings
+
+  @param  entry_point
+  @param  count
+  @param  p_binding_numbers
+  @param  p_set_numbers
```

Comment:
are these 3 right?

Reply:
same for the functions below: are the parameter names right?

Reply:
Good point, the documentation for the original functions I copied from is also wrong, I'll fix those too.